src/app/models/reservation.js

```diff
@@ -0,0 +1,26 @@
+import Backbone from 'backbone';
+
+const Reservation = Backbone.Model.extend({
+  baseURL: 'https://ada-backtrek-api.herokuapp.com/trips',
+  url: function() {
+    return this.baseURL + '/' + this.attributes.tripId + '/' + 'reservations';
```

Comment:
You could also specify a `urlRoot` function.
```
urlRoot() {
  return `${this.baseURL}/${this.get('tripId')/+reservations`;
},
```

Note the string interpolation.